src/components/FarmsList/SelectedFarm/SelectedFarm.tsx

```diff
@@ -0,0 +1,185 @@
+          </Button>
+        </Box>
+        <Typography className={classes.greenText}>
+          Unclaimed rewards:
+          <Typography display='inline' component='span' className={classes.value}>
+            <AnimatedNumber
+              value={value}
+              duration={300}
+              formatValue={(value: string) => Number(value).toFixed(2)}
+            />
+            {currency}
+          </Typography>
+        </Typography>
+      </Grid>
+      {activeValue === 'stake' ? (
+        <Grid className={classes.infoContainer}>
+          <Box className={classes.boxLeft}>
+            <Typography className={classes.infoText}>
+              Total Staked:
+              <Typography display='inline' component='span' className={classes.value}>
+                <AnimatedNumber
+                  value={value}
+                  duration={300}
+                  formatValue={(value: string) => Number(value).toFixed(2)}
+                />
+                {currency}
+              </Typography>
+            </Typography>
+            <Typography className={classes.infoText}>
+              APY:
+              <Typography display='inline' component='span' className={classes.value}>
+                <AnimatedNumber
+                  value={apy}
+                  duration={300}
+                  formatValue={(value: string) => Number(value).toFixed(1)}
+                />
+                %
+              </Typography>
+            </Typography>
+            <Typography className={classes.infoText}>
+              Liquidity:
+              <Typography display='inline' component='span' className={classes.value}>
+                $
+                <AnimatedNumber
+                  value={liquidity}
+                  duration={300}
+                  formatValue={(value: string) => Number(value).toFixed(0)}
+                />
+              </Typography>
+            </Typography>
+          </Box>
+          <Box className={classes.boxRight}>
+            <Typography className={classes.infoText}>
+              Total Staked:
+              <Typography display='inline' component='span' className={classes.value}>
+                <AnimatedNumber
+                  value={value}
+                  duration={300}
+                  formatValue={(value: string) => Number(value).toFixed(2)}
+                />
+                {currency}
+              </Typography>
+            </Typography>
+            <Typography className={classes.infoText}>
+              Total Staked:
+              <Typography display='inline' component='span' className={classes.value}>
+                <AnimatedNumber
+                  value={value}
+                  duration={300}
+                  formatValue={(value: string) => Number(value).toFixed(2)}
+                />
+                {currency}
+              </Typography>
+            </Typography>
+            <Typography className={classes.infoText}>
+              Total Staked:
```

Comment:
why is this value repeated many times? consult with someone, what should be shown there